.gitignore

```diff
@@ -36,6 +36,9 @@ ENV/
 env.bak/
 venv.bak/
 
+# Development credentials
+credentials/
```

Comment:
I would prefer this file to stand in the application directory (see comments below).